napari/_vispy/_text_utils.py

```diff
@@ -0,0 +1,59 @@
+from typing import Tuple
+
+import numpy as np
+from vispy.scene.visuals import Text
+
+
+def update_text(
+    text_values: np.ndarray,
+    coords: np.ndarray,
+    anchor: Tuple[str, str],
+    rotation: float,
+    color: np.ndarray,
+    size: float,
+    ndisplay: int,
+    text_node: Text,
+):
+    """ Update the vispy text node with the current text and display parameters
```

Comment:
```suggestion
    """Update the vispy text node with the current text and display parameters.
```

@Carreau, help us! 😂

Reply:
I am really looking forward to your docstring formatter, @Carreau! Until then, sorry @jni - it's not intentional!

Reply:
you can run it here: https://github.com/carreau/minirst still highly in progress.